tools/cleanup-excludelists.sh

```diff
@@ -4,7 +4,7 @@ set -eo pipefail
 
 cd $(dirname "${BASH_SOURCE[0]}")/..
 
-TEMP=$(mktemp --tmpdir cleanup-excludelists-XXXXXXXX)
+TEMP=$(mktemp cleanup-excludelists-XXXXXXXX)
```

Comment:
If you remove the `--tmpdir`, GNU mktemp it will create the file in the current directory rather than /tmp. Sigh.

It looks like the best cross-platform option is to change the call to something like
```suggestion
TEMP=$(mktemp "${TMPDIR:-/tmp}/cleanup-excludelists-XXXXXXXX")
```

Reply:
I need to change this to 
```suggestion
TEMP=$(mktemp "${TMPDIR:-/tmp/}cleanup-excludelists-XXXXXXXX")
```
with the trailing slash assumed. `TMPDIR` for me was preset to include the trailing slash.

Would that work for you? For me on OSX, if the extra slash is added, the `trap` fails.

Reply:
I'm starting to dislike OS X, why does the trap fail if there's a double slash?

That proposal will break if TMPDIR is set to something that _doesn't_ end in a slash.

```suggestion
TMPDIR="${TMPDIR:-/tmp}"
TEMP=$(mktemp "${TMPDIR%/}/cleanup-excludelists-XXXXXXXX")
```

Reply:
> I'm starting to dislike OS X, why does the trap fail if there's a double slash?

I'm already there :). It didn't delete the file for me. I didn't dig in.

In any event, this one is a winner for me. Accepting your suggestion and awaiting your approval.